Software/bsw/static/COM/CanIf/inc/CanIf.h

```diff
@@ -376,7 +382,7 @@ typedef struct {
 
 				/*This container contains the configuration (parameters) of an adressed CAN controller by
 				an underlying CAN Driver module. This container is configurable per CAN controller.*/	
-				CanIfCtrlCfgType CanIfCtrlCfgObj;
+				CanIfCtrlCfgType CanIfCtrlCfgObj;//It should be CanIfCtrlCfgType CanIfCtrlCfgObj[MAX_CONFIGURABLE_CAN_CONTROLLERS];
```

Comment:
Yes and the naming of MAX_CONFIGURABLE_CAN_CONTROLLERS must match the one in CAN DRIVER

Reply:
Sure InshaAllah